libs/designsystem/src/lib/components/chart/configs/annotations.config.ts

```diff
@@ -33,4 +33,6 @@ export const CHART_ANNOTATIONS_CONFIG: AnnotationTypeRegistry = {
   point: {
     backgroundColor: 'initial',
   },
+  label: { content: () => null },
+  polygon: { backgroundColor: 'initial' },
```

Comment:
Are these changes safe? They stick out compared to the other objects.  

You are the chart champ after all 🙇, but I noticed these changes are from the commits before you took over this branch, so just wanted to make sure.